spec/core/ics-002-client-semantics/README.md

```diff
@@ -660,228 +573,47 @@ to allow governance mechanisms to perform these actions
 ```typescript
 function updateClient(
   id: Identifier,
-  header: Header) {
-    clientType = provableStore.get(clientTypePath(id))
-    abortTransactionUnless(clientType !== null)
-    clientState = provableStore.get(clientStatePath(id))
-    abortTransactionUnless(clientState !== null)
-    clientType.checkValidityAndUpdateState(clientState, header)
+  clientState: ClientState,
```

Comment:
I think we should retrieve the clientState from the store here instead of passing it as an argument, i.e.,
```
clientState = provableStore.get(clientStatePath(id))
abortTransactionUnless(clientState !== null)
```